### PR TITLE
Add tx address and id for tx history table

### PR DESCRIPTION
**Pull request by liviavinci**

### CODE REVIEW

**Comment by liviavinci** [comment]
screen shot 2018-03-23 at 2 58 09 am
screen shot 2018-03-23 at 2 58 22 am
screen shot 2018-03-23 at 2 58 53 am

File: src/components/TransactionHistoryAddressAndID/address.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          <div>
+            {transaction.senderAddress}
+          </div>
+        </div>
+      </TableCell>
+    );
+  }
+}
+
+TransactionHistoryAddress.propTypes = {
```
Comment (dwalintukan) [inline src/components/TransactionHistoryAddressAndID/address.js]: move in class as `static propTypes`

File: src/components/TransactionHistoryAddressAndID/id.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          <div>
+            {transaction.txid}
+          </div>
+        </div>
+      </TableCell>
+    );
+  }
+}
+
+TransactionHistoryID.propTypes = {
```
Comment (dwalintukan) [inline src/components/TransactionHistoryAddressAndID/id.js]: move to class as `static propTypes`

File: src/scenes/Activities/scenes/EventHistory/index.js [hunk 3]
```diff
@@ -222,45 +224,64 @@ class EventHistory extends React.Component {
...
+          }
+        </TableCell>
+      </TableRow>
+    );
+    result[1] = (
+      <TableRow key={`txaddr-${transaction.txid}`} selected>
+        <TransactionHistoryAddress transaction={transaction} />
+        <TableCell /><TableCell />
+        <TransactionHistoryID transaction={transaction} />
+        <TableCell /><TableCell /><TableCell /><TableCell />
```
Comment (dwalintukan) [inline src/scenes/Activities/scenes/EventHistory/index.js]: are these for spacing?

Comment (liviavinci) [reply src/scenes/Activities/scenes/EventHistory/index.js]: yup

File: src/components/TransactionHistoryAddressAndID/address.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import { TableCell } from 'material-ui/Table';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { withStyles } from 'material-ui/styles';
+
+import styles from './styles';
+
+class TransactionHistoryAddress extends React.PureComponent {
```
Comment (alex-cory) [inline src/components/TransactionHistoryAddressAndID/address.js]: This should be a stateless component if we're not using state.

const TransactionHistoryAddress = ({ classes, transaction }) => (
  ...
)

**Pull request merged**